packages/zoe/src/contractFacet/contractFacet.js

```diff
@@ -452,23 +450,26 @@ export function buildRootObject(powers, _params, testJigSetter = undefined) {
 
     // First, evaluate the contract code bundle.
     const contractCode = evalContractBundle(bundle);
-    contractCode.catch(err => {
-      // Remove to suppress Node.js's UnhandledPromiseRejectionWarning
-      throw err;
-    });
+    // Don't trigger Node.js's UnhandledPromiseRejectionWarning.
+    // This does not suppress any error messages.
+    contractCode.catch(() => {});
```

Comment:
Elsewhere this uses `catch(_ => {})`. Is it significant that it's different that this catch clause?

At least file a ticket ticket to move all these wild catch clauses until a call to a single helper. Ideally, add your own helper here for the moment so they can all be found and replaced when we have a system-wide helper.

Reply:
> Elsewhere this uses catch(_ => {}). Is it significant that it's different that this catch clause?

Nope, it's the same. 

I filed an issue to move the catch clauses to a single helper (#2039)